### PR TITLE
Fixed some LGTM alerts

### PR DESCRIPTION
**Pull request by imsolost**
Hello there. I was reviewing your project and I found some [alerts on LGTM.com](https://lgtm.com/projects/g/uber/luma.gl/alerts/?mode=tree). I decided to fix a few of them, but I definitely encourage you to check out the remaining alerts.

I know this may not be the type of contribution you normally receive, but I figured I could help clean up the codebase.

(Full disclosure: I'm part of the team behind LGTM.com)

Let me know if you have any questions.

### CODE REVIEW

File: modules/core/src/webgl-context/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export {createGLContext, destroyGLContext, resizeGLContext, pollGLContext} from './context';
 
-export {withParameters, resetParameters} from './context-state';
+export {withParameters, resetParameters, setParameters} from './context-state';
```
Comment (ibgreen) [inline modules/core/src/webgl-context/index.js]: Interesting, I wonder why this was changed? 
- Does LGTM just try to make sure all exports are forwarded? 
- Or is it a result of a multi-file/multi-depth analysis that concluded this was imported somewhere else?

Comment (imsolost) [reply modules/core/src/webgl-context/index.js]: The [LGTM alert I found](https://lgtm.com/projects/g/uber/luma.gl/alerts/?mode=tree&ruleFocus=10030087) for this issue showed that setParameters was not a function and had a type of undefined. From that I was able to trace the error to the index file which was not exporting setParameters.

Comment (ibgreen) [reply modules/core/src/webgl-context/index.js]: That sounds great, definitely very useful

**Comment by imsolost** [comment]
@ibgreen The answer to all of your questions is yes! LGTM is free for open source projects and you can use it as a tool for [automatic code review and continuous integration](https://lgtm.com/projects/g/uber/luma.gl/ci/). If you decide to set it up for your project, think about adding the LGTM badge to your project as well.

**Pull request merged**